CONTRIBUTING.md

```diff
@@ -0,0 +1,243 @@
+# Contribuire al progetto
+
+Grazie per l'interesse mostrato nei confronti del progetto ! 🎉
+
+Cittadinanza Digitale è un progetto open source:
+sono benvenuti contributi, suggerimenti e richieste di nuove caratteristiche.
+
+Quel che segue è un insieme di linee guida per poter contribuire,
+scrivendo codice o integrando la documentazione,
+all'evoluzione delle componenti che costituiscono il frontend
+(App mobile) e il backend (API) di Cittadinanza Digitale.
+
+## Come riportare un bug o proporre una feature
+
+Se hai domande o proposte ti suggeriamo di condividerle sul forum dedicato,
+nella sezione [Piattaforme abilitanti](https://forum.italia.it/c/piano-triennale/piattaforme-abilitanti).
+
+In alternativa puoi trovarci su Slack nel 
+[canale #io-devel di developers.italia.it](https://slack.developers.italia.it/).
+
+# Risorse
+
+Lo sviluppo dell'app e del suo backend avviene su [GitHub](http://github.com).
+
+Prima di iniziare ti consigliamo di leggere i README contenuti nella directory radice
+dei repository elencati in questa sezione. Potrai trovare:
+
+- la descrizione dell'architettura del backend (e le scelte che ne hanno portato la definizione)
+- la descrizione dettagliata del funzionamento del software e l'interazione tra le componenti
+- le procedure per sviluppare localmente e le dipendenze da installare
+
+## Repository GitHub
+
+[Repository principale con la descrizione dell'infrastruttura](https://github.com/teamdigitale/digital-citizenship)
+
+[Sorgenti delle API di notifica / preferenze (Azure Functions)](https://github.com/teamdigitale/digital-citizenship-functions)
+
+[Sorgenti dell'App mobile](https://github.com/teamdigitale/italia-app)
+[Sorgenti del backend dell'App](https://github.com/teamdigitale/italia-backend)
+
+[Sorgenti del proxy per le API di PagoPa](https://github.com/teamdigitale/italia-pagopa-proxy)
+[Sorgenti del client API PagoPA](https://github.com/teamdigitale/italia-pagopa-api)
+
+[Utilità per l'onboarding nel developer portal](https://github.com/teamdigitale/digital-citizenship-onboarding)
+
+[Sorgenti del sito web di presentazione dell'App](https://github.com/teamdigitale/italia-landing)
+
+## Siti web
+
+Esistono due siti Web relativi al progetto:
+
+[io.italia.it](http://io.italia.it): presentazione dell'app mobile,
+rivolto ai Cittadini che vorranno utilizzarla.
+
+[teamdigitale.github.io/digital-citizenship](https://teamdigitale.github.io/digital-citizenship): 
```

Comment:
Credo che confluirà tutto in `io.italia.it` per cui questo possiamo toglierlo per ora - @matteodesanti